{"uid":"ea0a8a75840bf743","name":"Comparing Dashboard data with List data for Vendors","fullName":"TestInventDashboard.TestInventDashboard#test_get_vendor_rpt","historyId":"2e8b555e209bd4cd53c6e61aec4b4e7f","time":{"start":1683886098903,"stop":1683886110984,"duration":12081},"status":"failed","statusMessage":"AssertionError: False != True : Verification Failed to mismatch title.","statusTrace":"self = <TestsCases.TestInventDashboard.TestInventDashboard testMethod=test_get_vendor_rpt>\n\n    @allure.title(f\"Comparing Dashboard data with List data for Vendors\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_get_vendor_rpt(self):\n        self.LoginMethod.nativelogin(self.driver, \"frugal@latido.com.np\", \"Test@123\")\n        try:\n            self.assertEqual(self.InventDash.get_full_vendor_rpt(), True, msg=\"Verification Failed to mismatch title.\")\n        # Checking if assertion failed\n        except AssertionError as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\", attachment_type=AttachmentType.PNG)\n            self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\n>           raise e\n\nTestInventDashboard.py:63: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTestInventDashboard.py:58: in test_get_vendor_rpt\n    self.assertEqual(self.InventDash.get_full_vendor_rpt(), True, msg=\"Verification Failed to mismatch title.\")\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: False != True : Verification Failed to mismatch title.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestInventDashboard","time":{"start":1683886067683,"stop":1683886067683,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: False != True : Verification Failed to mismatch title.","statusTrace":"self = <TestsCases.TestInventDashboard.TestInventDashboard testMethod=test_get_vendor_rpt>\n\n    @allure.title(f\"Comparing Dashboard data with List data for Vendors\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_get_vendor_rpt(self):\n        self.LoginMethod.nativelogin(self.driver, \"frugal@latido.com.np\", \"Test@123\")\n        try:\n            self.assertEqual(self.InventDash.get_full_vendor_rpt(), True, msg=\"Verification Failed to mismatch title.\")\n        # Checking if assertion failed\n        except AssertionError as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\", attachment_type=AttachmentType.PNG)\n            self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\n>           raise e\n\nTestInventDashboard.py:63: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTestInventDashboard.py:58: in test_get_vendor_rpt\n    self.assertEqual(self.InventDash.get_full_vendor_rpt(), True, msg=\"Verification Failed to mismatch title.\")\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: False != True : Verification Failed to mismatch title.","steps":[],"attachments":[{"uid":"c0d235a25d398356","name":"Failed_for_customer","source":"c0d235a25d398356.png","type":"image/png","size":152369},{"uid":"1779cc13501ab7e2","name":"log","source":"1779cc13501ab7e2.txt","type":"text/plain","size":154}],"parameters":[],"stepsCount":0,"hasContent":true,"attachmentsCount":2,"shouldDisplayMessage":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestInventDashboard::0","time":{"start":1683886111000,"stop":1683886111000,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Inventory Dashboard"},{"name":"suite","value":"TestInventDashboard"},{"name":"subSuite","value":"TestInventDashboard"},{"name":"host","value":"0day"},{"name":"thread","value":"16516-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"TestInventDashboard"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"ea0a8a75840bf743.json","parameterValues":[]}